### PR TITLE
docs: update 5.2.0 changelog

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | chore: update changelog |
| 🇨🇳 Chinese | - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
https://github.com/ant-design/ant-design/releases/tag/5.2.0 记得也补一下

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ timeline: true
   - 🐞 修复选择列固定时滚动会被其他单元格遮盖的问题。[#39940](https://github.com/ant-design/ant-design/pull/39940) [@kiner-tang](https://github.com/kiner-tang)
   - 🐞 修复排序/筛选的表格的固定列背景色透明导致显示异常问题。[#39012](https://github.com/ant-design/ant-design/pull/39012) [@kiner-tang](https://github.com/kiner-tang)
   - 💄 修复 Table filter 时边框样式问题。[#39938](https://github.com/ant-design/ant-design/pull/39938) [@JarvisArt](https://github.com/JarvisArt)
+  - 💄 Table hover 样式回滚到 v4 [#40469](https://github.com/ant-design/ant-design/pull/40469)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:54]: 别这么写，写具体解决了什么问题

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ timeline: true
   - 🐞 修复选择列固定时滚动会被其他单元格遮盖的问题。[#39940](https://github.com/ant-design/ant-design/pull/39940) [@kiner-tang](https://github.com/kiner-tang)
   - 🐞 修复排序/筛选的表格的固定列背景色透明导致显示异常问题。[#39012](https://github.com/ant-design/ant-design/pull/39012) [@kiner-tang](https://github.com/kiner-tang)
   - 💄 修复 Table filter 时边框样式问题。[#39938](https://github.com/ant-design/ant-design/pull/39938) [@JarvisArt](https://github.com/JarvisArt)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:53]: 这条可以删了，都被回滚了

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: done~

File: components/timeline/index.zh-CN.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ demo:
 - 当有一系列信息需按时间排列时，可正序和倒序。
 - 需要有一条时间轴进行视觉上的串联时。
 
-<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以提供具有更好的性能和使您在应用程序中能编写更简单代码潜力。 与此同时，我们弃用了旧的用法，我们将在 antd 6.0 中删除它。"></Alert>
+<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以提供更好的性能，和使您在应用中能编写更简单代码的潜力。与此同时，我们弃用了旧的用法，我们将在 antd 6.0 中删除它。"></Alert>
```
Comment (MadCcc) [inline components/timeline/index.zh-CN.md:18]:
```suggestion
<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以获得更好的性能，使您能在应用中编写更简单的代码。与此同时，我们弃用了旧的用法，我们将在下一个 major 版本中删除它。"></Alert>
```
感觉像机翻中文。。

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md]: done~

File: components/timeline/index.zh-CN.md (hunk 3 above)
Comment (li-jia-nan) [reply components/timeline/index.zh-CN.md:18]: 我也感觉是机翻

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ timeline: true
   - 🐞 修表头过滤器在分组标题情况下失效的问题。[#40463](https://github.com/ant-design/ant-design/pull/40463) [@roman40a](https://github.com/roman40a)
   - 🐞 修复选择列固定时滚动会被其他单元格遮盖的问题。[#39940](https://github.com/ant-design/ant-design/pull/39940) [@kiner-tang](https://github.com/kiner-tang)
   - 🐞 修复排序/筛选的表格的固定列背景色透明导致显示异常问题。[#39012](https://github.com/ant-design/ant-design/pull/39012) [@kiner-tang](https://github.com/kiner-tang)
-  - 💄 修复 Table filter 时边框样式问题。[#39938](https://github.com/ant-design/ant-design/pull/39938) [@JarvisArt](https://github.com/JarvisArt)
+  - 💄 优化 Table hover 样式[#40469](https://github.com/ant-design/ant-design/pull/40469)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:53]:
```suggestion
  - 💄 优化 Table 组件 hover 样式，修复边框异常问题。[#40469](https://github.com/ant-design/ant-design/pull/40469)
```

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ timeline: true
   - 🐞 Fix the problem that the header filter is invalid in the case of group headers. [#40463](https://github.com/ant-design/ant-design/pull/40463) [@roman40a](https://github.com/roman40a)
   - 🐞 Fix selection column cover by other cell when fixed. [#39940](https://github.com/ant-design/ant-design/pull/39940) [@kiner-tang](https://github.com/kiner-tang)
   - 🐞 Fix Sorted/Filtered table fixed column transparent background unreadable. [#39012](https://github.com/ant-design/ant-design/pull/39012) [@kiner-tang](https://github.com/kiner-tang)
-  - 💄 Fix border style problem when Table filter. [#39938](https://github.com/ant-design/ant-design/pull/39938) [@JarvisArt](https://github.com/JarvisArt)
+  - 💄 Optimization Table hover style [#40469](https://github.com/ant-design/ant-design/pull/40469)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:53]:
```suggestion
  - 💄 Optimization Table hover style to fix problems with border. [#40469](https://github.com/ant-design/ant-design/pull/40469)
```

File: components/timeline/index.zh-CN.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ demo:
 - 当有一系列信息需按时间排列时，可正序和倒序。
 - 需要有一条时间轴进行视觉上的串联时。
 
-<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以提供具有更好的性能和使您在应用程序中能编写更简单代码潜力。 与此同时，我们弃用了旧的用法，我们将在 antd 6.0 中删除它。"></Alert>
+<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以获得更好的性能，使您能在应用中编写更简单的代码。与此同时，我们弃用了旧的用法，我们将在下一个 major 版本中删除它。"></Alert>
```
Comment (MadCcc) [inline components/timeline/index.zh-CN.md:18]:
```suggestion
<Alert message="5.2.0版本之后，我们提供了更简单的用法 &lt;Timeline items={[...]} /&gt; 以获得更好的性能，使您能在应用中编写更简单的代码。与此同时，我们弃用了旧的用法，并且将在下一个 major 版本中删除它。"></Alert>
```

File: components/timeline/index.zh-CN.md (hunk 3 above)
Comment (yoyo837) [reply components/timeline/index.zh-CN.md]: 我按英文版, 以自己的解读人工写的. 😭

**Comment by li-jia-nan** [comment]
> https://github.com/ant-design/ant-design/releases/tag/5.2.0 记得也补一下

done～

**Comment by li-jia-nan** [comment]
求+1

**Review by zombieJ — APPROVED** [review]

**Pull request merged**